### PR TITLE
bump to sylius 1.7

### PR DESCRIPTION
**Pull request by pfz**
| Q               | A
| --------------- | -----
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | yes
| License         | MIT

### CODE REVIEW

**Comment by patrick477** [comment]
thank you

File: composer.json [hunk 1]
```diff
@@ -9,15 +9,15 @@
     "description": "CMS plugin for Sylius applications.",
     "license": "MIT",
     "require": {
-        "php": "^7.2",
+        "php": "^7.3",
         "league/csv": "^9.1",
         "friendsofsymfony/ckeditor-bundle": "^1.1|^2.0",
         "stefandoorn/sitemap-plugin": "^1.0",
-        "sylius/sylius": "^1.4"
+        "sylius/sylius": "^1.7"
```
Comment (oallain) [inline composer.json:16]: Hi @pfz and @patrick477 

Why this plugin no longer compatible with versions prior to 1.7 of Sylius? And PHP 7.2 ?

Comment (pfz) [reply composer.json:16]: Hi, "knplabs/gaufrette" was locked to 0.5 and prevented the use of 1.7+ versions of Sylius, which require php>=7.3, and gaufrette > 0.7.

Upgrade requirement is an easy solution for me I brought from my forked version of this cms. Although I consider moving up to php7.3 is good practice for me, as 7.2 is EOL in 6 months.
(see https://www.php.net/manual/en/migration73.incompatible.php)

Comment (oallain) [reply composer.json:16]: Sylius support 1.6 right now.
So ok for PHP but you can reduce Sylius?

Comment (patrick477) [reply composer.json:16]: sylius 1.6 is supported in the version 2.3.*

Comment (pfz) [reply composer.json:16]: @oallain The gaufrette dependency problem appears from sylius 1.7. No luck here.

Comment (oallain) [reply composer.json:16]: Okay, thanks

**Pull request merged**